### ISSUE TITLE
Incorrect newlines for binary expression with === operator

### ISSUE THREAD

**Issue by dsherret** [dsherret/dprint#39]
See here:

https://dprint.dev/playground/#code/MYewdgzgLgBFCm0CWYDmMC8dFRegZPjAIQLQB0ZuaAKgBZIQDyArlALABQMPvfPGQSSp5ilHHgDcQA/config/N4IgNglgdgpg6hAJgFwBYgFwA4AMAaEaRGKZBFdDAFgIFcBnGAFQEMAjezAMxbEYMYBbCAGEA9mDFROGZACdaMAdADmYGAEVaY5DBk8+SkPJYRIUFeMGCWMkLABuMOSDqMAQnJYBjPZhAA7qgkAHI6AMqq6gAy0DCuIGxevgAKYvQQyBBS-rAAHsixsACSXAASLBaqCRkW6u5iiACeaRlZORggNtDIplAJ+cjipHISAGKSAa2Z2f2dg0XxBGIADs4syGJy0+1z9jAFiwlcW74AsrRgWYsAgnIqtIIkyPq8-CAncueX13EpLF4nro5K9DAQSI8ACIwbxgAEbWYAOiegjYznCKx81U63VIfQSANGATGtCg3l2AFE8is5HoMlJEQwYP9aaRgow7LjetAQABfIA

Basically, `===` should not be put on a newline like that.

```ts
const testing = testing && !test.testingThisOut
    === !testing!.testing;
```
